### ISSUE TITLE
Horizon, HI, and pull requests

### ISSUE THREAD

**Issue by bleckett** [scott-vincent/instrument-panel#23]
Hi Scott - I've a couple of questions for you. 

1. Horizon (AI) control
 - I noticed the knob is remmed out. Is it because you ran out of pins? I was going to add it back in, but noticed that it doesn't sync up with FS2020. Is it something you're looking into or should I investigate?

2. HI - not smooth
 - I noticed that the indicator is not smooth when setting a new heading. It looks like the value is rounding or resetting, causing the heading indicator in the instrument panel to stutter. I'm just speculating as I haven't gone into debugging anything yet. Again, is it something you're aware of or should I take a look?

3. Pull Requests
- Lastly, would you like us to request/pull out a fork or do you prefer us just sending you simple code snips to integrate on your own?

Thanks.

**Comment by scott-vincent** [comment]
Yes, I ran out of inputs on the Pi so only the essential knobs are wired up currently. Bruce has a To Do list and one thing he is hoping to do is move all the knob config out into a separate settings file which will make things a lot easier.

I seem to remember the heading indicator does move in 5 degree increments but also the instrument displays the value coming back from the SIm rather than the value you actually set meaning there is a slight delay. I'm sure some work could be done there to smooth it out. The problem is it can be set either by the HI knob or by the Autopilot heading knob or by the in-game cockpit knob so it can be adjusted from multiple places.

Merge Requests are fine. I'm happy to merge fixes, and also enhancements if it's generally beneficial to everybody or if there is a setting to enable/disable if it's not something everybody would find useful, e.g. I've replaced ADF by a RadioComm instrument in my panel but I don't enable this by default.

**Comment by bleckett** [comment]
Great. Thanks. I'll see what I can do about getting the Artificial Horizon to sync up with MSFS3020's panel. 

For the HI, I simply ended up changing line 173 in your hi.cpp file...
from ...  
`If (adjust != 0) {`
to
`If ((adjust >=4) || (adjust <=-4)) {`

Works great so far, but I need to test it with changes coming from FS2020. 

I also removed the shadowing just for the indicator so that the ticks stay clear under the marker. Maybe add an override in the Jason file for individual instrument shadowing/deshadowing? (LOL - more work!)

Anyway - once again... great job.
